k8s/client.go

```diff
@@ -1084,6 +1085,23 @@ func (c *Client) GetHelmState(ctx context.Context, namespace string, secretName
 	}, nil
 }
 
+// GetHelmValues is the function for cilium cli sysdump to collect the helm values from the release directly
+func (c *Client) GetHelmValues(_ context.Context, releaseName string) ([]byte, error) {
+	client := action.NewGetValues(c.HelmActionConfig)
+	vals, err := client.Run(releaseName)
+	if err != nil {
+		return nil, fmt.Errorf("unable to retrive the helm value from the release %s: %w", releaseName, err)
+	}
+
+	valuesBuf := new(bytes.Buffer)
+	encodeErr := output.EncodeYAML(valuesBuf, vals)
```

Comment:
follow the conversation from here

https://github.com/liyihuang/cilium-cli/commit/f093eec1c5fee350d8b900d62341b926da4932b8#r123316401


If that's the case, why the following can be reused with a new declare?


https://github.com/liyihuang/cilium-cli/blob/f093eec1c5fee350d8b900d62341b926da4932b8/k8s/client.go#L1067-L1077

Reply:
If there are _any_ new names on the left side of a `:=`, then the `:=` is allowed.